src/Tools/ExternalAccess/FSharp/Internal/Editor/FSharpNavigationBarItemService.cs

```diff
@@ -48,9 +48,9 @@ public void NavigateToItem(Document document, NavigationBarItem item, ITextView
                 var workspace = document.Project.Solution.Workspace;
                 var navigationService = workspace.Services.GetService<IFSharpDocumentNavigationService>();
 
-                if (navigationService.CanNavigateToPosition(workspace, document.Id, span.Start))
+                if (navigationService.CanNavigateToPosition(workspace, document.Id, span.Start, virtualSpace: 0, cancellationToken))
                 {
-                    navigationService.TryNavigateToPosition(workspace, document.Id, span.Start);
+                    navigationService.TryNavigateToPosition(workspace, document.Id, span.Start, virtualSpace: 0, options: null, cancellationToken);
```

Comment:
Use the extension method you added?

Reply:
this is IFSharpDocumentNavigationService, which doesn't have those extensions :)